package.json

```diff
@@ -9,7 +9,14 @@
         "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
         "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
         "start": "webpack --watch",
-        "test": "mocha-webpack test/* --recursive --require test/.setup"
+        "test": "mocha-webpack test/* --recursive --require test/.setup",
+        "precommit": "lint-staged"
+    },
+    "lint-staged": {
+        "*.js": [
+            "prettier --write --tab-width 4",
+            "git add"
```

Comment:
Does this mean all `.js` files will be committed automatically? If so, we don't want that.

Reply:
No, it runs prettier on .js files staged for commit. Don't merge this branch yet, I'll add pre-commit hooks for .vue and .json